### ISSUE TITLE
Disagreement with styler and lintr about long lines. Rstudio gives a better format. 

### ISSUE THREAD

**Issue by maikol-solis** [tidyverse/style#213]
Hi! 

I have this problem. Suppose you have this long line in your code:

```
value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)
```

Checking it with `styler`, returns the unmodified string. 
```r
styler::style_text("value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)")
value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)
```
However, `lintr` is complaining about the length of the line
```
[line_length_linter]: Lines should not be more than 80 characters. This line is 100 characters.	
```

In my opinion, Rstudio gives a better format: 
```r
value <-
  my_function(
    arg0 = 0,
    arg1 = 1,
    arg2 = 2,
    arg3 = 3,
    arg4 = 4,
    arg5 = 5,
    arg6 = 6,
    arg7 = 7
  )
```

What can I do to make `styler` behave like Rstudio in this case? 

Best.

**Comment by maikol-solis** [comment]
Sorry, I posted it in the wrong repo.
